Assets/Resources/AI_Server.txt

```diff
@@ -1,3 +1,3 @@
-AI_API_URL=http://91.107.198.4:8000/
+AI_API_URL=http://127.0.0.1:8001
```

Comment:
This would have to be a public URL, maybe we can also switch to the domain now? repository.wekit-ecs.com:8000 should be the public master release, and :8001 should be for develop (currently isn't)?